packages/benchmarks/src/textToDriver/makeDatabases/loadBraintrustDatasets.ts

```diff
@@ -0,0 +1,108 @@
+import { initDataset } from "braintrust";
+import { EJSON } from "bson";
```

Comment:
> Rather than using bson as a standalone library, can we use the version bundled with mongodb/exported from mongodb-rag-core?

i feel like i tried this and it didn't work but can take another look

Reply:
yep this worked